index.js

```diff
@@ -49,19 +49,25 @@ program
         var resp;
         while(true) {
             resp = await sequencer.tryContribute(sessionID);
-            if(resp.status === 200) {
+            if(resp.status == 200) {
                 if(resp.error != null) {
-                    const lobby = await sequencer.getStatus()
-
-                    console.log(`- ${lobby.lobby_size} contributors are waiting...`);
                     console.log(`${resp.error} - retry after ${RETRY_SEC} seconds`);
-
                     await sleep(RETRY_SEC);
                     continue
                 }
                 break;
             }
 
+            if(resp.status == 400) {
+                console.log(resp.msg);
+                console.log(`Retry after ${RETRY_SEC} seconds`);
+                await sleep(RETRY_SEC);
+                continue;
+            } else {
+                console.log(`Error ${resp.status}`);
+                return;
+            }
+
```

Comment:
I think it is the same code as 71-79.
`===` is the only difference.
Is there a reason why you added it?

Reply:
Removed. This duplication may occured by fault to fix the confliction from 1ff86b.